src/js/core/UpdateStream.ts

```diff
@@ -0,0 +1,145 @@
+import { Observable } from "rxjs/Observable";
+import compareVersions from "compare-versions";
+import { request, RequestResponse } from "@dcos/http-service";
+import { BehaviorSubject } from "rxjs";
+
+import UserSettingsStore from "#SRC/js/stores/UserSettingsStore";
+import Util from "#SRC/js/utils/Util";
+import { SAVED_STATE_KEY } from "#SRC/js/constants/UserSettings";
+
+export const UpdateStreamType: symbol = Symbol("UpdateStreamType");
+const CHECK_DELAY: number = 24 * 60 * 60 * 1000; // Once every 24 hours
```

Comment:
```suggestion
const CHECK_DELAY = 24 * 60 * 60 * 1000; // Once every 24 hours
```

Reply:
Not necessary, but it would be removing unnecessary information, making it more concise

Reply:
Then we have to remove the type of all other `const`s like `DCOS_UI_VERSION_SETTING` as well.